config/routes.rb

```diff
@@ -3,4 +3,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  shallow do
+    resources :users, only: [:index, :show] do
+      resources :posts, only: [:index]
```

Comment:
- [ ] in this case, for the posts only the index endpoint is available please make sure the show endpoint is also available.

Reply:
Thanks for the observation. It might have escaped me.